packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -44,61 +44,64 @@ export type FeaturesContextProviderProps = {
   flags: IFlags | undefined;
 };
 
+const getFeatures = (flags: IFlags): FeaturesData => {
+  const steps = getFeatureValue(Features.OnboardingSteps, flags);
+  const onboardingSteps = (steps?.split?.('/') || []) as OnboardingStep[];
+
+  return {
+    flags,
```

Comment:
Should we even provide the raw flags? I think we should consider ignoring them completely after we parse them. This will force us to parse and add proper support for each flag. cc @rebelchris

Reply:
Yes, that is true. This was just included before for backwards compatibility. We can probably create a ticket to cleanup all legacy implementations.

Reply:
cc @rebelchris